src/main.ts

```diff
@@ -90,7 +99,11 @@ function installRuntimeDependencies(extension: vscode.Extension<any>): Promise<v
         })
         .then(() => {
             installationStage = 'touchLockFile';
-            return util.touchLockFile();
+            return util.touchInstallFile(util.InstallFileType.Lock);
+        })
+        .then(() => {
+            installationStage = 'deleteBeginFile';
+            return util.deleteInstallFile(util.InstallFileType.Begin)
```

Comment:
Should this be in the catch block (or using the temp file stuff) so that we delete the begin file no matter what?